### ISSUE TITLE
Unexpected termination when syntax error is introduced to active preview in VSCode

### ISSUE THREAD

**Issue by kopeckyf** [christianvoigt/argdown#195]
Dear all,

I've come across a replicable crash, which is related to a specific usage pattern of Argdown.

**Version**: Visual Studio Code 1.51.1 with Argdown extension 1.5.3
**Error**: `Extension host terminated unexpectedly.`

**Steps to reproduce**:

1. Open a new Argdown document in Visual Studio Code.
2. Enter a very simple sketch, e.g.:

```
# Ausschnitt der Debatte aus der Entscheidung des BerlVerfGH

[Gebot, sich ständig in der eigenen Wohnung aufzuhalten]: Es ist verhältnismäßig, dass sich zur Eindämmung der Covid-19-Pandemie grundsätzlich in der eigenen Wohnung aufgehalten werden muss.

<Argument 1>: Staatlicher Eingriff verboten
    -> [Gebot, sich ständig in der eigenen Wohnung aufzuhalten]
```

3. Enter the preview mode with this button:
![image](https://user-images.githubusercontent.com/10461729/100987321-a82a5580-354e-11eb-99e7-46cb8c0a2162.png)

4. Then attempt to enter premises for `<Argument 1>`. The crash occurs after two empty lines are entered after the heading:

```
# Ausschnitt der Debatte aus der Entscheidung des BerlVerfGH

[Gebot, sich ständig in der eigenen Wohnung aufzuhalten]: Es ist verhältnismäßig, dass sich zur Eindämmung der Covid-19-Pandemie grundsätzlich in der eigenen Wohnung aufgehalten werden muss.

<Argument 1>: Staatlicher Eingriff verboten



    -> [Gebot, sich ständig in der eigenen Wohnung aufzuhalten]
```

**Work around:** One can click "Restart extension host" and the problem appears to go away.

**Possible solution:** Only update the preview on explicit user request, e.g. by pressing the button in the top bar.

Best,
Felix

**Comment by christianvoigt** [comment]
Thanks for the exemplary bug report. It took some time to find the time for Argdown maintenance, but I am on it now.

**Comment by christianvoigt** [comment]
fixed in v1.7.0
